app/models/concerns/itemizable.rb

```diff
@@ -67,19 +67,25 @@ def total
     validates_associated :line_items
   end
 
-  def line_items_quantities
-    line_items.inject(Hash.new) do |hash, line_item|
-      hash[line_item.id] = OpenStruct.new(quantity: line_item.quantity, item_id: line_item.item_id)
-      hash
-    end
-  end
-
   def value_per_itemizable
     line_items.sum(&:value_per_line_item)
   end
 
+  def to_a
```

Comment:
Whichever of you made this know that I love you.

Reply:
We were pairing, so you are legally required to love both of us.